D1673/P1673.md

```diff
@@ -8122,7 +8117,7 @@ void scaled_matvec_2(
 }
 
 // z = 7.0 times the transpose of A, times y
```

Comment:
```suggestion
// z = 7.0 * (the transpose of A) * y
```



LWG wanted `*` here instead of "times," for consistency with the comments above.  I actually would prefer using "times" in all the comments in this example, but `*` is fine.